### ISSUE TITLE
Low accuracy on Jester dataset with pre-trained model

### ISSUE THREAD

**Issue by stefanobini** [okankop/Efficient-3DCNNs#40]
Hi, I have downloaded you repository and the Jester dataset, then I have followed the istructions contained in the "README.md" file to preprocess the dataset in order to obtain the files required by the framework and finally I have ran the system in test mode both on the validation and test set with the pre-trained model ResNeXt-101 (jester_resnext_101_RGB_16_best.pth) and MobileNetv2 1.0x (jester_mobilenetv2_1.0x_RGB_16_best.pth). But the performance is completly different: about 3% on validation set and 10% on test set. So, I asked me if you can share witrh me the way to reproduce your results.

**Comment by pestrstr** [comment]
Hi Stefano, have you solved the issue?
I'm having the same issue too. I cannot reproduce the results of this paper.
I have tested pre-trained weights on test videos and by closely looking at the output of the softmax layer, the model seems to always produce 0 as predicted label. I'm wondering if I'm testing it with wrong transforms / normalization criteria.
